go.mod

```diff
@@ -30,6 +30,8 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require github.com/google/btree v1.1.3
```

Comment:
Move this to the other requires?

Reply:
`go mod tidy` does this, I don't know offhand why the new section but this something I manually changed

Reply:
yeah I've been usually manually moving these requires in the single larger block

Reply:
Oh didn't know `go mod tidy` will let that be, done